sphinx/templates/quickstart/conf.py_t

```diff
@@ -78,7 +78,7 @@ language = {{ language | repr }}
 exclude_patterns = [{{ exclude_patterns }}]
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'sphinx'
+pygments_style = None
```

Comment:
Note: the original setting was born at 7d9721dd85 (v0.2). So this setting is a part of Sphinx itself even if it's a bug.